ciao-controller/types/types.go

```diff
@@ -714,3 +716,34 @@ type QuotaUpdateRequest struct {
 type QuotaListResponse struct {
 	Quotas []QuotaDetails `json:"quotas"`
 }
+
+func NewTenantHardwareAddr(ip net.IP) net.HardwareAddr {
```

Comment:
Also you need comments of the form // NewTenantHardwareAddr .... (failed on go linting)